main.nf

```diff
@@ -65,6 +65,7 @@ def helpMessage() {
       --max_allele_freq [float]         Maximum allele frequency threshold for filtering variant calls (Default: 0.8)
       --save_align_intermeds [bool]     Save the intermediate BAM files from the alignment steps (Default: false)
       --save_mpileup [bool]             Save MPileup files generated during variant calling (Default: false)
+      --skip_markdups [bool]            Skip picard MarkDuplicates step (Default: false)
```

Comment:
are you sure about the params?
why not `skip_markduplicates`?

Reply:
I did have that initially!! But then I put in another param called `--keep_dups` in another PR so thought it would be better for consistency with that :) Also, saves have a super long param name.

Reply:
True, the params name is long, but it is accurate, and understandable by everyone.
And not sure that `keep_dups` is a good name either.

Reply:
Yeah, not massively fussed about it to be honest. Could change it back to `--skip_markduplicates` 👍 What would your suggestion for `--keep_dups` be then? Have been using that for a while in the `atacseq` and `chipseq` pipelines now without any issue. Thats not to suggest its the right choice but just that I havent had any confusion so far 😉

Reply:
See https://github.com/nf-core/viralrecon/pull/94/commits/f7004dd083f523674b0b5b7b3d98176b8887b43e

Reply:
`--save_duplicates` ?